src/app/component/AddToCart/AddToCart.component.js

```diff
@@ -52,14 +52,16 @@ export default class AddToCart extends PureComponent {
             product: { type_id },
             isLoading,
             buttonClick,
+            disabledButtonClick,
```

Comment:
If new prop is added, it must be declared in prop types. I do not see this being done here.

Reply:
Try configuring ESLint - it will start telling you if you did something wrong :)